app/javascript/gamesFactory.js

```diff
@@ -74,15 +75,19 @@ angular.module('dappChess').factory('games', function (crypto, navigation,
    *  },
    *  ended: <boolean>,
    *  pot: <number>,
+   *  turnTime: <number>,
    *  timeoutStarted: <date>,
-   *  timeoutState: <{-2,-1,0,1,2}>
+   *  timeoutState: <{-1,0,1}>
```

Comment:
@graup: From what I can see in the contract, this is wrong. Any particular reason you changed it back from the (correct) state in develop?